notifier/notifier_test.go

```diff
@@ -185,6 +187,54 @@ func TestHandlerSendAll(t *testing.T) {
 	}
 }
 
+func TestHandlerQueuing(t *testing.T) {
+	dropRegex, _ := config.NewRegexp("drop")
+	renameRegex, _ := config.NewRegexp("rename")
```

Comment:
Never drop errors, in general but especially not in Go. Either check and `t.Fatalf` or use the `config.MustNewRegexp` constructur. The latter can directly be inlined in the options initialization below.